_posts/2024-05-28-pyos-newsletter-june-2024.md

```diff
@@ -0,0 +1,112 @@
+
+## <i class="fa-solid fa-envelope-open"></i> Save the Date: pyOpenSci's Open Source Fall Festival
+<figure>
+    <a href="/images/blog/2024/may/oss-fall-festival.png">
+    <img src="/images/blog/2024/may/oss-fall-festival.png" style="max-width:100%" alt="Abstract shapes on a white background with the text: Save the Date. pyOpenSci Open Source Fall Festival. September 28 & 29, 2024. Entirely online! Follow us on socials for more info. fosstodon.org/@pyOpenSci linkedin.com/company/pyopensci pyopensci.org.">
+    </a>
+</figure>
+We love a good get-together, and our Open Source Fall Festival is no exception! Held entirely online using [Gather](https://www.gather.town/), our goals is to bring together members of the Python community that create open source tools with the members of the Python community who use the tools. In building this festival, our focus is on a grassroots, community-led event with some big unconference vibes. So mark your calendars, and save the date for September 28--29th, 2024!
+
+## <i class="fa-solid fa-hand-holding-heart"></i> Python packages for everyone!
+pyOpenSci's [Open Peer Review process](https://www.pyopensci.org/about-peer-review/index.html) is experiencing an [unprecedented number of Python package submissions](https://github.com/pyOpenSci/software-submission/issues!) We're excited to see so many wonderful packages that are helping scientists help make the world a better place being submitted for review. Our open peer review process facilitates scientists getting credit and recognition for the work they’ve invested in developing scientific Python tools. The peer review process also supports scientists in finding vetted and maintained software, which drives their open science workflows.
+
+You can submit your package for review (or pre-submission inquiry to determine if it's in scope) by [submitting an issue](https://github.com/pyOpenSci/software-submission/issues/new/choose) to [our software-submission repository](https://github.com/pyOpenSci/software-submission). And because all of our reviews are open, [you can see our review process in action](https://github.com/pyOpenSci/software-submission).
+
+We're also accepting applications for [package reviewers](https://docs.google.com/forms/d/e/1FAIpQLSeVf-L_1-jYeO84OvEE8UemEoCmIiD5ddP_aO8S90vb7srADQ/viewform). We welcome reviewers from a diversity of background and with varying levels of Python technical expertise.
+
+Some of the basic things that we look for in a review include:
+* Familiarity with using the Python programming language.
+* Ability to evaluate a Python package for usability and documentation quality.
+* Ability to provide a technical review of Python package structure and code quality / approach to solving the problems that the package seeks to address.
+
+We like to have a mix of technical and usability focus in our reviews so it’s ok if you don’t have all of the above skills!
```

Comment:
Maybe a call to action - not sure if you're ready to review on your own? check out our mentorship program <link??> or something like that.

Reply:
love it! I'll add it.